### ISSUE TITLE
ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by dpungaliya** [desittisagar/Web-Scraper#1]
It shows this error when i simply copy your main.py code and paste it,and run it(using Jupyter)

ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-5c65bef39a53> in <module>
----> 1 import requests
      2 from requests import get
      3 from bs4 import BeautifulSoup
      4 import numpy as np
      5 import pandas as pd

ModuleNotFoundError: No module named 'requests'

Please help me resolve this issue

**Comment by desittisagar** [comment]
You have to install requests and for that type 'pip install requests' or 'python -m pip install requests'. Before doing this you must install pip.

**Comment by dpungaliya** [comment]
> You have to install requests and for that type 'pip install requests' or 'python -m pip install requests'. Before doing this you must install pip.

okay,thank you! i'll try this

**Comment by dpungaliya** [comment]
> You have to install requests and for that type 'pip install requests' or 'python -m pip install requests'. Before doing this you must install pip.

I did this,not its showing the error-

ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-5c65bef39a53> in <module>
      1 import requests
      2 from requests import get
----> 3 from bs4 import BeautifulSoup
      4 import numpy as np
      5 import pandas as pd

ModuleNotFoundError: No module named 'bs4'

​so,which library needs to be installed?

**Comment by desittisagar** [comment]
Yeah, you have to install all the dependencies mentioned in the code...
Pip install bs4
Pip install numpy
Pip install pandas
Try these...

**Comment by dpungaliya** [comment]
> Yeah, you have to install all the dependencies mentioned in the code...
> Pip install bs4
> Pip install numpy
> Pip install pandas
> Try these...

okay,i'll do this,Thank you!

**Comment by dpungaliya** [comment]
> Yeah, you have to install all the dependencies mentioned in the code...
> Pip install bs4
> Pip install numpy
> Pip install pandas
> Try these...

got it!Thanks
